Doc/README.md

```diff
@@ -0,0 +1,134 @@
+# fontTools Documentation
+
+The fontTools project documentation updates continuously on Read the Docs as the project source changes.  
+
+The documentation can be found at **TODO - RTD URL when pushed and available**.
+
+## Contents
+
+- [How to Build Local Documentation](#how-to-build-local-documentation)
+- [Contributing to the fontTools Documentation](#contributing-to-the-documentation)
+- [Documentation License](#documentation-license)
+
+## How to Build Local Documentation
+
+### Install Dependencies
+
+You must have a Python 3 interpreter and the `pip` Python package manager installed on your system to build the fontTools documentation.  
+
+Use the following command to install the necessary Python build dependencies:
+
+```
+$ pip3 install sphinx sphinxcontrib-napoleon sphinx-rtd-theme
+```
+
+### Clone the fontTools repository
+
+Use the following command to clone the fontTools repository with `git`:
+
+```
+$ git clone https://github.com/fonttools/fonttools.git
+```
+
```

Comment:
I need to add venv instructions here.  OK to use the same venv setup instructions that are used on the README or add a link to those instructions?  Any plan to change the dev env setup approach?

Reply:
don't overcomplicate yourself. Building documentation from source is an advanced user topic. People interested in doing that already know what to do. Just keep it short and simple.

Reply:
like, you don't need to tell people how to git clone fonttools

Reply:
will simplify.  thank you!

Reply:
fixed in https://github.com/fonttools/fonttools/pull/1333/commits/07f81d968f4e4086817a95921471741671f58f30